plugins/git/git.plugin.zsh

```diff
@@ -103,20 +103,20 @@ function gccd() {
 compdef _git gccd=git-clone
 
 alias gcl='git clone --recurse-submodules'
-alias gclean='git clean -id'
+alias gclean='git clean --interactive -d'
 alias gpristine='git reset --hard && git clean -dffx'
 alias gcm='git checkout $(git_main_branch)'
 alias gcd='git checkout $(git_develop_branch)'
-alias gcmsg='git commit -m'
+alias gcmsg='git commit --message'
 alias gco='git checkout'
 alias gcor='git checkout --recurse-submodules'
-alias gcount='git shortlog -sn'
+alias gcount='git shortlog --summary -n'
```

Comment:
```suggestion
alias gcount='git shortlog --summary --numbered'
```
https://git-scm.com/docs/git-shortlog#Documentation/git-shortlog.txt---numbered

Reply:
Oops, there's no reason that was just a mistake! I used some regexes to fix most flags, then came back for the grouped ones (i.e. `-sn`) by hand. Good catch! I'll fix these.